SmartDeviceLink/SDLDateTime.h

```diff
@@ -0,0 +1,73 @@
+//  SDLDateTime.h
+//
+
+#import "SDLRPCStruct.h"
+
+@interface SDLDateTime : SDLRPCStruct
+
+- (instancetype)initWithHour:(UInt8)hour minute:(UInt8)minute second:(UInt8)second millisecond:(UInt16)millisecond day:(UInt8)day month:(UInt8)month year:(UInt16)year timezoneMinuteOffset:(UInt8)timezoneMinuteOffset timezoneHourOffset:(int)timezoneHourOffset;
+
+/**
+ * @abstract Milliseconds part of time
+ *
+ * Required, Integer 0 - 999
```

Comment:
Note that in the proposal, all these were changed to be optional. https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0004-sendlocation-updates.md